src/plugins/data/common/search/aggs/buckets/histogram.ts

```diff
@@ -20,14 +20,16 @@
 import { get } from 'lodash';
 import { i18n } from '@kbn/i18n';
 
-import { KBN_FIELD_TYPES, UI_SETTINGS } from '../../../../common';
+import { isAutoInterval, KBN_FIELD_TYPES, UI_SETTINGS } from '../../../../common';
```

Comment:
Why dont we also import it from `_interval_options` ?

Reply:
Thank you, my IDE decided to add it into the wrong place. Fixed